Source/Android/jni/CMakeLists.txt

```diff
@@ -55,17 +55,17 @@ PRIVATE
 
 target_include_directories(main
 PRIVATE
-    ${CMAKE_SOURCE_DIR}/Externals/libadrenotools/include
-    ${CMAKE_SOURCE_DIR}/Externals/VulkanMemoryAllocator/include
+    ${CMAKE_CURRENT_SOURCE_DIR}/Externals/libadrenotools/include
+    ${CMAKE_CURRENT_SOURCE_DIR}/Externals/VulkanMemoryAllocator/include
```

Comment:
This now points at `[dolphin]/Source/Android/jni/Externalslibadrenotools/include`, and so on.

Reply:
Okay, I'll revert my larger set of changes and keep the scope down.